schema.sql

```diff
@@ -330,6 +330,15 @@ CREATE TABLE IF NOT EXISTS `tile_store` (
   FOREIGN KEY (`house_id`) REFERENCES `houses` (`id`) ON DELETE CASCADE
 ) ENGINE=InnoDB;
 
+CREATE TABLE IF NOT EXISTS `live_casts` (
```

Comment:
This is actually a very good point, although it would also need a column to check if said player is in fact live casting. Having a separate table looks cleaner but saving a little memory may be worth it in the long run.

And if you're wondering about that extra column; it's for Tibia 11 and Flash client once they become more common, since they would use a webbased authentication and wouldn't be able to smoothly get that information from the engine itself.